### ISSUE TITLE
Adding a glossary

### ISSUE THREAD

**Issue by jhdalek55** [uptane/deployment-considerations#83]
During our 1/5 meeting it was suggested we add a glossary/definition of terms page to Deployment Best Practices. While such a page does exist in the Standard, it makes sense for it to appear in Best Practices so users do not need to refer back to the Standard. Also, adding it to this repo means a quick and easy reference page on the web site as well.

**Comment by jhdalek55** [comment]
@plapczyn not necessarily. I think the definitions referred to #85 might be a bit more in-depth than what this glossary calls for. In effect, this glossary would be the same as the "Terminology" section of the Standard. The Definitions called for in Issue #85 will likely require more elaboration, and perhaps examples/case studies. In short, they will probably be more than a one or two sentence definition.

**Comment by jhdalek55** [comment]
@pattivacek I think to add this page and the list of standards and regulations for Issue #84  I will need two markdown files...unless you can think it would make more sense to incorporate both these things into the the introduction? What like an opinion....

**Comment by pattivacek** [comment]
I agree, these sound like two pages to me. I think the list of relevant standards and regulations fits fairly well with the Deployment Best Practices, but I think the glossary might be a better fit for the "Basics" section of the website. It's probably fine wherever it ends up, but it seems less about deployment and more general.

**Comment by jhdalek55** [comment]
@iramcdonald had mentioned a glossary should be in the Deployment document, so while having it on the website as well is a good idea, it needs to  be in the document. Initially, to was just going to transfer over the existing definitions from the standard, unless the consensus is that we need a more detailed vocabulary for these pages. 
@pattivacek can you create the markdown files for me? That's the one thing I don't know how to do. I can then begin to fill in the content.

**Comment by pattivacek** [comment]
> @iramcdonald had mentioned a glossary should be in the Deployment document, so while having it on the website as well is a good idea, it needs to be in the document.

No problem, that's fine.

> @pattivacek can you create the markdown files for me? That's the one thing I don't know how to do. I can then begin to fill in the content.

Done, see https://github.com/uptane/deployment-considerations/pull/97.

**Comment by jhdalek55** [comment]
Thanks...I fill these in later today or tomorrow.

**Comment by jhdalek55** [comment]
I prepared this, but accidentally committed it directly to master. Since this is a duplicate of what is already in the Standard, I don't think this is a big deal. The only reason I wanted to make this a pull request is I wanted to know if there was more we needed to add other than what was already in the Standard.

**Comment by jhdalek55** [comment]
Opened PR# 99 to address additional comments.

Lois

**Comment by jhdalek55** [comment]
To administer @iramcdonald's recommendation to move the glossary from the Standard to the Deployment pages we need to remove the text all under terminology from the main file of the Standard and then replace it with a pointer towards the Deployment "Glossary" pages. While I could do the edit, I'm not sure about setting up the pointer. Could someone take care of this? Meanwhile I am going to ensure the full glossary is set up as some text was removed when I thought this would a duplicate of the page in the Standard. Both need to be completed before we can close this issue. Thanks.

**Comment by jhdalek55** [comment]
I am going to close this issue and open one on the Standard repo to remove the glossary from the Standard and add a pointer to the glossary on the Deployment pages.

**Comment by jhdalek55** [comment]
Closed via PR #99.  Opened a new issue on the Standard repo to remove the text there and add a pointer.
